app/assets/stylesheets/partials/_fonts.scss

```diff
@@ -0,0 +1,51 @@
+@import 'https://fonts.googleapis.com/css?family=Lato:300,400,400i,700,900';
+@import 'https://fonts.googleapis.com/css?family=Roboto+Mono:400,500';
+@import "partials/variables";
+
+body {
+  color: $black;
+  font-family: $font-stack;
+  font-size: 16px;
+  line-height: 1.5;
+}
+
+::selection {
+  background: $black-80;
+  color: white;
+}
+::-moz-selection {
+  background: $black-80;
+  color: white;
+}
+
+a,
+.link {
+  color: $green;
```

Comment:
you can start changing these colors to be more arctic-friendly! also, test around and try to remove the styles that are unused from this file, _modal.scss, and _layouts.scss (aka don't break the code when you take them out)